lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -393,7 +393,13 @@ def insert_at_position(position)
         end
 
         def update_positions
-          old_position = send("#{position_column}_was") || bottom_position_in_list + 1
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
```

Comment:
In fact, let's get rid of checking the version altogether, and rely on `respond_to?` for this.  Why check version, When we can duck type.